.eslintrc.json

```diff
@@ -67,6 +67,7 @@
     "react/require-default-props": "off",
     "no-useless-constructor": "off",
     "sonarjs/no-small-switch": "off",
+    "sonarjs/no-unused-vars": "off",
```

Comment:
I don't think we should disable this rule globally. What was causing the issue?

Reply:
Oh I see. I assume we'll still get linting errors if we have unusued variables. Since `@typescript-eslint/no-unused-vars` is disabling the linting errors for only `...rest` and variables/args starting with `_`.